crowbar_framework/app/models/backup.rb

```diff
@@ -32,16 +33,11 @@ class Backup < ActiveRecord::Base
       message: "allows only letters and numbers"
     }
 
-  validates :version,
-    presence: true
-
-  validates :size,
-    presence: true
```

Comment:
agree that the commit message should explain it. it is not validated, as the values are set by crowbar, so we assume they are valid.